src/components/forms/validator.ts

```diff
@@ -100,8 +100,12 @@ export const mustBeEthereumContractAddress = memoize(
   },
 )
 
-export const minMaxLength = (minLen: number, maxLen: number) => (value: string): ValidatorReturnType =>
-  value.length >= +minLen && value.length <= +maxLen ? undefined : `Should be ${minLen} to ${maxLen} symbols`
+export const minMaxLength = (minLen: number, maxLen: number) => (value: string): ValidatorReturnType => {
+  const testValue = value || ''
```

Comment:
This isn't needed. The type of value is always a string.

Reply:
I also thought that, but at some point this value is undefined and breaks the app. Could only find this as fast fix. I can add a FIXME here if you consider it

Reply:
We weren't seeing this before because this validator was always used with `required` so it was chasing this cases